server/src/test/java/org/apache/uniffle/server/storage/LocalStorageManagerTest.java

```diff
@@ -209,21 +218,33 @@ public void testInitializeLocalStorage() {
 
     // case2: when candidates exist, it should initialize successfully.
     conf.set(
-        ShuffleServerConf.RSS_STORAGE_BASE_PATH, Arrays.asList("/a/rss-data", "/tmp/rss-data-1"));
```

Comment:
It will fail to create `/a/rss-data` directory on Github Linux machines due to lacking POSIX permissions. This is the reason why the test might fail on other machines.